src/frontend/src/catalog/table_catalog.rs

```diff
@@ -401,6 +403,7 @@ impl TableCatalog {
             initialized_at_epoch: self.initialized_at_epoch.map(|epoch| epoch.0),
             created_at_epoch: self.created_at_epoch.map(|epoch| epoch.0),
             cleaned_by_watermark: self.cleaned_by_watermark,
+            create_status: PbTableStatus::Creating.into(),
```

Comment:
Note that `to_prost` is not only used for streaming job creation, but also used for `alter_table_column`, which will replace the whole catalog for target table. Making the status default to `Creating` is acceptable to me, but you need to set it to `Create` in meta side not only for creation but also for table altering. You can do it in this PR or in later one.

Reply:
Thanks for mentioning it. Will add it here.

Reply:
https://github.com/risingwavelabs/risingwave/pull/12342